DynamoLeagueBlazor/Shared/Enums/Position.cs

```diff
@@ -14,7 +14,34 @@ public abstract class Position : SmartEnum<Position>
 
         public abstract int[] PerYearContractPriceTable();
 
-        // TODO: Revisit ContractOption
+
+        public IEnumerable<ContractOption> CalculateContractYearOptions(int bidValue)
+        {
+            var yearsToContract = 0;
+            foreach (var price in PerYearContractPriceTable())
+            {
+                yield return new ContractOption((DateTime.Now.Year + yearsToContract++), Math.Max(bidValue, price));
+            }
+        }
+        public int GetContractValue(int contractValue, int yearContractExpires)
```

Comment:
is this used? If so I’ve got some follow up comments

Reply:
Yes it is

Reply:
The 2nd variable in the param list should probably be named contractedToYear too, no?

Reply:
NumberPfYears isn’t bad - do you like yearDifference or something with the word “difference” better to denote that’s what it is?

Reply:
On line 30 combine the conditional so that there is one return